vgl/pyvgl.cxx

```diff
@@ -374,6 +377,9 @@ void wrap_vgl_vector_3d(py::module &m, std::string const& class_name)
     .def(py::self * T())
     .def(T() * py::self)
     ;
+
+    // overloads for non-member functions using vgl_vector_3d
+    m.def("cross_product", py::overload_cast<const vgl_vector_3d<T>&, const vgl_vector_3d<T>&> (&cross_product<T>));
```

Comment:
After talking with Dan, the weird way cross_product was implemented might've been the only way to do it in an earlier version of pybind; there are better ways now.

I updated vgl_vector_3d so cross_product can be called by vgl.cross_product and marked the old way as deprecated.